tools/src/test/resources/1.6/invalid-component-ref-1.6.json

```diff
@@ -10,12 +11,6 @@
       "name": "acme-library",
       "version": "1.0.0"
     },
-    {
```

Comment:
this was a duplicate entry, which is just not relevant for the JSON implementation.